backend/src/main/java/ai/verta/modeldb/project/ProjectServiceImpl.java

```diff
@@ -750,12 +715,15 @@ public void getProjects(
               .addAllProjects(projects)
               .setTotalRecords(projectPaginationDTO.getTotalRecords())
               .build();
-      saveAuditLogs(
+      Workspace workspace =
+          roleService.getWorkspaceByWorkspaceName(userInfo, request.getWorkspaceName());
+      saveAuditLog(
           Optional.of(userInfo),
           ModelDBServiceActions.READ,
-          projects,
+          "",
```

Comment:
There is a ModelDBConstants.EMPTY_STRING constant

Reply:
Ohh ok. updating

Reply:
updated